components/Chips/tests/snapshot/MDCChipViewLayoutSnapshotTests.m

```diff
@@ -259,6 +269,33 @@ - (void)testSelectedChipImagePaddingAllPositiveValuesForLargeSelectedImageLTR {
   [self generateSnapshotAndVerifyForView:self.chipView];
 }
 
+- (void)testUnselectedChipImagePaddingAllPositiveValuesForSmallSelectedImageLTR {
+  // Given
+  self.chipView.selectedImageView.image =
+      [UIImage mdc_testImageOfSize:CGSizeMake(8, 8)
+                         withStyle:MDCSnapshotTestImageStyleDiagonalLines];
+
+  // When
+  self.chipView.imagePadding = UIEdgeInsetsMake(10, 20, 30, 40);
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.chipView];
+}
+
+- (void)testSelectedChipImagePaddingAllPositiveValuesForSmallSelectedImageLTR {
+  // Given
```

Comment:
I'm noticing a vertical misalignment between the image on the left and the title+box on the right (in this and otehr snapshots). Is that a bug?

Reply:
Do you mean the `imageView`'s vertical alignment compared to that of the `titleLabel` and `accessoryView`?  This test is adjusting the `imagePadding` value, would should manipulate the space around the `imageView`. Since `top` and `left` are less than `bottom` and `right`, the imageView should be pushed up and toward the leading edge of the Chip.